Packet++/header/IgmpLayer.h

```diff
@@ -326,13 +330,13 @@ class IgmpV3QueryLayer : public IgmpLayer
 
 	/**
 	 * A constructor that allocates a new IGMPv3 membership query
-	 * @param[in] multicastAddr The multicast address to set. This is an optional parameter and has a default value of IPv4Address#Zero
+	 * @param[in] multicastAddr The multicast address to set. This is an optional parameter and has a default value of unspecified IPv4 address

```

Comment:
Can you add `unspecified/zero` instead of just `unspecified`?

Reply:
Done